src/Employer/Employer.Web/Orchestrators/Part1/TitleOrchestrator.cs

```diff
@@ -37,11 +38,25 @@ public async Task<TitleViewModel> GetTitleViewModel(string employerAccountId)
             var vm = new TitleViewModel
             {
                 PageInfo = new PartOnePageInfoViewModel(),
-                ShowReturnToMALink = dashboard == null || !dashboard.CloneableVacancies.Any()
             };
+            GetBackLink(dashboard, vm);
             return vm;
         }
 
+        private static void GetBackLink(EmployerDashboard dashboard, TitleViewModel vm)
```

Comment:
This is named incorrectly. Its setting not getting.

Can we get rid of this.  Can we put back the `ShowReturnToMALink` and just have 2 links on the view. One with a `asp-show` the other with `asp-hide`. That way a FE dev can change the link text if needed.